src/Repository/LinkBlockRepository.php

```diff
@@ -213,6 +265,7 @@ public function createTables()
             "CREATE TABLE IF NOT EXISTS `{$this->dbPrefix}link_block_shop` (
     			`id_link_block` int(10) unsigned NOT NULL auto_increment,
     			`id_shop` int(10) unsigned NOT NULL,
+                `position` int(10) unsigned NOT NULL default '0',
```

Comment:
Same as previous about tabs and spaces

Reply:
do you mind if he does that in a PR dedicated to linting ?  (applying cs fixer on all PHP files, and fixing these tab issues...

Reply:
as it wasn't that bad I decided to use cs fixer in this PR

Reply:
nice ! 👍